userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -148,6 +167,9 @@ Kill Bill Administrator (or Kill Bill Admin):: The person responsible for instal
 
 == O
 
+[[overdue_system]]
+Overdue System (or Dunning):: A Kill Bill module that defines the flow that the system must follow when an account is overdue (that is, has an unpaid balance). It is used in conjunction with the <<payment_retry_mechanism, payment retry system>>.
```

Comment:
```suggestion
Overdue System (or Dunning):: A Kill Bill module that defines the flow that the system must follow when an account is overdue (that is, has an unpaid balance). It is used in conjunction with the <<payment_retry_mechanism, payment retry module>>.
```

Reply:
^^We have not standardized this yet. I think of "system" as the overall umbrella, with "modules" or "sub-systems" making it up.